index.js

```diff
@@ -308,18 +316,29 @@ async function smartContractInitialization() {
    log(`deploy Uniswap2 mainnet`)
    const router = await deployUniswap2(wallet)
    log(`deploy Uniswap2 sidechain`)
-   await deployUniswap2(sidechainWallet)
+   const uniswapRouterSidechain = await deployUniswap2(sidechainWallet)
    
    tx = await token.approve(router.address, amt_token)
    //await tx.wait()
    tx = await token2.approve(router.address, amt_token2)
    await tx.wait()
    log(`addLiquidity Uniswap2 mainnet`)
    tx = await router.addLiquidity(token.address, token2.address, amt_token, 
-    amt_token2, 0, 0, wallet.address, futureTime)
+   amt_token2, 0, 0, wallet.address, futureTime)
+   
+   let cf = new ContractFactory(Uniswap2Adapter.abi, Uniswap2Adapter.bytecode, wallet)
```

Comment:
would be better to use `const` and different variable names, only use `let` if that can't be done for some reason (code branching reasons usually)

Reply:
there is also a ticket in TODO to clean up and lint the whole repo. things like this will be solved by that ticket, but I haven't got around to it yet..